### DISCUSSION TITLE
Debug output in pm2io.from_interchange_format

### DISCUSSION THREAD

**Discussion by rgieseke** [pik-primap/primap2#86] [General]
Is this debug output intentional or leftover from development? (I can probably suppress it with changing log levels.)

https://github.com/pik-primap/primap2/blob/becf899220c2a047d8802527acc97e9e07bbd696/primap2/pm2io/_interchange_format.py#L321

**Comment by mikapfl** [comment]
That's intentional. It is pretty helpful (the expected array shapes are often rather unexpected :wink: ) and at level debug, it is not expected to be seen by users generally.

**Reply by rgieseke** [reply]
Well, I did see it :-)
What do you mean by users? Someone who uses pre-generated dataset?

**Reply by rgieseke** [reply]
(I agree that is helpful as i start to wrap my around the data formats :-))

**Reply by mikapfl** [reply]
Hm, true, kind of weird that debug seems to be the default level, I would have thought that without config debug messages are hidden.

...if you don't want to see it, you probably have to call `loguru.configure()` with the right parameters. That's more work than it should be, I think, but possible.

**Reply by rgieseke** [reply]
Sounds reasonable (it is helpful so i probably keep it, any log output makes me just think that something's wrong), thx! :-)

**Reply by mikapfl** [reply]
When something's wrong, we'll log `error` or `critical` and the log message will be red and maybe blinking. (-;
